lib/Constants.ts

```diff
@@ -438,7 +438,7 @@ const _commandClassMap = {
 	0xf0: 'non_interoperable',
 }
 export function commandClass(cmd: number): string {
```

Comment:
You're basically duplicating the `CommandClasses` enum from `zwave-js` - not sure if there isn't a better way?

Reply:
yeah but the problem is that using the zwave-js command class names could create breaking changes in mqtt topics